src/config.h.cmake.in

```diff
@@ -181,7 +181,7 @@
 #cmakedefine SIZEOF_VOID_P ${SIZEOF_VOID_P}
 
 /* Define to 1 if you have the ANSI C header files. */
-#cmakedefine STDC_HEADERS
+#cmakedefine STDC_HEADERS 1
```

Comment:
Thanks for the PR. I see a small issue: if someone defines `STDC_HEADERS` as 0 the change would still trigger a warning, no?

Reply:
How would someone do that?  In my case, STDC_HEADERS is defined by the system headers.  See this from my bug report:

```
In file included from /shaka-packager/packager/third_party/glog/src/demangle.h:73,
                 from /shaka-packager/packager/third_party/glog/src/demangle.cc:39:
/shaka-packager/build/packager/third_party/glog/config.h:184: warning: "STDC_HEADERS" redefined
 #define STDC_HEADERS
 
In file included from /usr/include/c++/8/cstdio:41,
                 from /shaka-packager/packager/third_party/glog/src/demangle.cc:37:
/usr/include/x86_64-linux-gnu/c++/8/bits/c++config.h:1377: note: this is the location of the previous definition
 #define STDC_HEADERS 1
```

So STDC_HEADERS is defined in `/usr/include/x86_64-linux-gnu/c++/8/bits/c++config.h` for me.  I can't control the definition of it there, so I chose to make the definition in glog match, on the assumption that this was needed somehow to build glog.

But now that I look more closely, STDC_HEADERS is not actually referenced anywhere in glog.  Would you accept a PR that simply removes it from `config.h.cmake.in` entirely?

Reply:
If you can't control how `STDC_HEADERS` is defined, you need to guard its definition. I'd rather remove the definition altogether.

Reply:
I'll update the PR to remove it completely from glog.  It doesn't appear to be used or needed.

I dug into the history to make sure I wasn't missing something important.  It looks like the cmake definition was put in [7 years ago](https://github.com/google/glog/commit/3c49b93201b143ead7a6a3287338e9ae2955ee4d) when cmake support was added, and it has never been touched since.  It also appears that at the time, it was copied (slightly incorrectly) from an old autoconf-based header template.  That header no longer exists in the repo, and the macro appears to be an autoconf-specific macro that corresponds to "headers like stdio.h, etc exist".